cypress/e2e/case-details/handle-triggers-and-exceptions/edit-fields-with-exceptions/edit-asn.cy.ts

```diff
@@ -236,8 +247,10 @@ describe("ASN", () => {
   it("should display error when invalid ASN is entered", () => {
     loginAndVisit("/bichard/court-cases/0")
 
-    cy.get("#asn").type("asdf")
+    cy.get("#asn").clear()
+    cy.get("#asn").type("1101ZD01448754")
 
+    cy.get(".Defendant-details-table").find(".warning-icon").should("exist")
```

Comment:
Class name with capital letter? :(

Reply:
[5b6cebd](https://github.com/ministryofjustice/bichard7-next-ui/pull/647/commits/5b6cebd01d630b480f9b5b406c717fc0a5b91a7f)

Reply:
This is `styled-components` doing 🙃